alerts.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  alert_resource_group_name = "disposer-shared-${var.env}"
+}
+
+module "idam-user-disposer-action-group" {
+  source                 = "git@github.com:hmcts/cnp-module-action-group"
+  location               = "global"
+  env                    = var.env
+  resourcegroup_name     = local.alert_resource_group_name
+  action_group_name      = "${var.application_name}-${var.env}-ag"
+  short_name             = "dispr-alert"
```

Comment:
Short_name , it will be good to have disposer-alert as everywhere we used disposer as short version as well